.gitmodules

```diff
@@ -18,3 +18,6 @@
 [submodule "src/liblibc"]
 	path = src/liblibc
 	url = https://github.com/rust-lang/libc.git
+[submodule "src/doc/nomicon"]
+	path = src/doc/nomicon
+	url = https://github.com/rust-lang-nursery/nomicon
```

Comment:
I wonder if this book was official part of the rust-lang, why "downgrade" it to `rust-lang-nursery` instead of keeping a repo in `rust-lang` ?